lib/LaTeXML/Core/Document.pm

```diff
@@ -1842,9 +1843,11 @@ sub appendElementBox {
     setNodeBox($self, $p, $newbox);
     # AND, propogate to autoOpen'd ancestors due to same initial box (See appendTextBox)
     while (($p = $p->parentNode) && ($p->nodeType == XML_ELEMENT_NODE)
-      && $p->getAttribute('_autoopened')
-      && ((getNodeBox($self, $p) || '') eq $origbox)) {
-      setNodeBox($self, $p, $newbox); } }
+      && $p->getAttribute('_autoopened')) {
+        my $obox = getNodeBox($self,$p);
+        last if $obox && (($box eq $obox) || ($box eq ($obox->unlist)[-1]));
+        setNodeBox($self,$p, ($p ? List($obox,$box) : $box)); } }
```

Comment:
This fixes the 5 or 5 problem, but I found another box-related bug: my tikzcd example contains
```latex
A \arrow[r, "\phi\eta" near start, "\phi" near start, "\psi"',"\eta" near end] & {\hbox{\large B}}
```
The foreignObject around the B is much wider than the aqua box. Exactly twice as large (coincidence?).